packages/product-components/src/components/TopAssets/TopAssets.tsx

```diff
@@ -30,6 +30,11 @@ const Item = styled('button')<{ $isLast: boolean }>`
     gap: ${spacingsPx.xxs};
     flex-direction: column;
     padding: ${spacings.xs * 1.25}px ${spacings.xs * 1.5}px ${spacings.xs * 0.75}px;
+    transition: background-color 150ms ease-in-out;
+
+    &:hover {
+        background-color: ${({ theme }) => theme.backgroundTertiaryPressedOnElevation0};
```

Comment:
You are using `elevation=1` above, so I would expect this color: `backgroundTertiaryPressedOnElevation1`